OpenSourceOrg/dotOrg
# [Develop] Update/wide template changes width all element develop

#### Changes proposed in this Pull Request

* Text blocks , form blocks, etc.. , weren't occupying the new page template wide width cause their max-width was set in other css files. this PR fixes that throughout

#### Testing instructions

* On a page with the template "No page header - wide" template, add paragraph blocks , form blocks , events listing ,etc... , to the page content.
* Confirm all tehse blocks now take up the full width of the template.

Mentions #
